README.md

```diff
@@ -23,6 +23,10 @@ Specialized methods manage categories and files:
    those that are no longer needed.
 - `list_files()` lists remote files, their attributes, and Unix-style path
   representations of their hierarchical position in the category tree.
+- `list_files()` List remote tax rates with their attributes.
+- `list_tax_rates()` lists remote files, their attributes, and Unix-style path
+- `list_accounts()` List remote accounts with their attributes.
```

Comment:
`list_accounts()` should add a Unix-style path  representation of the hierarchical position in the category tree. 

Change to:
- `list_accounts()` list remote accounts with their attributes, and Unix-style path representation of their hierarchical position in the category tree.